exercises/Graphing-1-Python.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: exercise
 title: Graphing 1
+subtitle: Mammal Bidy Size Pt 3
```

Comment:
> Bidy -> Body

Done. Apparently I can't type body effectively. I'm real glad you have a
look before they go through.

On Mon, Jul 20, 2015 at 5:32 AM, Ethan White <notifications@github.com>
wrote:

> In exercises/Graphing-1-Python.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/150#discussion_r34982847>
> :
>
> > @@ -1,6 +1,7 @@
> >  ---
> >  layout: exercise
> >  title: Graphing 1
> > +subtitle: Mammal Bidy Size Pt 3
>
> "Bidy" -> "Body"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/150/files#r34982847>
> .
>